app/components/avo/index/resource_controls_component.rb

```diff
@@ -65,8 +65,8 @@ def singular_resource_name
   end
 
   def parent_resource
+    return @parent_resource unless @parent_resource.blank?
```

Comment:
Can we use `if @parent_resource.present?` ? I think it would make it more readable this way